01_Tutorial/01_Basics.md

```diff
@@ -207,16 +214,16 @@ Your users will want to create a catalog of musical pieces to practice. You will
 
 Expand of your `_init` function to the following:
 
-```diff
+```js
```

Comment:
Please leave this as a diff, otherwise LGTM

Reply:
Ok. Did that.